docs/design/specs/Ecma-335-Augments.md

```diff
@@ -535,6 +545,13 @@ Interfaces may define static virtual methods that get resolved at runtime based
 
 ### II.12.2 Implementing virtual methods on interfaces
 
+(Add a new bullet under the second bullet in the bulleted list below the first paragraph by mentioning
+that classes can implement abstract static virtual methods by implementing a derived interface providing
+the default implementation for the abstract static virtual method):
```

Comment:
>default implementation [](http://example.com/codeflow?start=4&length=22)

Is it appropriate to call implementations in a derived interface a "default implementation"? I usually use term "default implementation" for a method that is implemented at the point of its declaration.

Reply:
I.e. declared virtual vs. abstract

Reply:
I'm not 100% sure about the precise terminology here. I was under the impression that the term "default interface implementations" was supposed to emphasize the deviation from the traditional notion that interfaces are just abstract API contracts. In general I like your point so for now I deleted the word `default` from the formulation.